app/main.ipynb

```diff
@@ -56,33 +59,53 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
    "source": [
-    "# write your code here (for Python default syntax)"
+    "arr = [0] * 1000\n",
+    "size = sys.getsizeof(arr)\n",
+    "print(size)"
    ],
    "metadata": {
     "collapsed": false,
     "ExecuteTime": {
-     "end_time": "2023-07-03T12:02:41.673421Z",
-     "start_time": "2023-07-03T12:02:41.666277Z"
+     "end_time": "2024-11-07T18:10:48.291556Z",
+     "start_time": "2024-11-07T18:10:48.285562Z"
     }
-   }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "8056\n"
+     ]
+    }
+   ],
+   "execution_count": 2
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
-   "outputs": [],
    "source": [
-    "# write your code here (for numpy syntax)"
+    "numpy_arr = np.array(arr)\n",
+    "size = numpy_arr.nbytes\n",
+    "print(size)"
```

Comment:
To minimize memory usage for the NumPy array, consider using `numpy_arr = np.zeros(1000, dtype=np.uint8)` instead of converting from a list. This will ensure the array uses as little memory as possible, as required by the task.